### DISCUSSION TITLE
Tui-select Web Accessibility Question

### DISCUSSION THREAD

**Discussion by Joshua-Ryan** [taiga-family/taiga-ui#1043] [Q&A] [answered]
I have a request for dropdown accessibility (currently using the tui-select and the tui-data-list-wrapper), however for full accessibility (according to: https://w3c.github.io/aria-practices/examples/combobox/combobox-select-only.html) when any printable character is pressed, the options should be searched/limited. What I am wondering is anyone has run into this issue before and what their work around was.

**Comment by waterplea** [comment, marked as answer]
Select is non editable, like native select. You need ComboBox. As for filtering, there's no built-in mechanism, but it's easily implemented with FilterByInput pipe:
https://taiga-ui.dev/components/combo-box#wrapper
